src/exchange/resource/resource.html

```diff
@@ -69,41 +69,45 @@
-                                <span class="label label-info"
+                                <span class="oui-status oui-status_info"
                                       data-translate="exchange_tab_ACCOUNTS_state_DELETING"
                                       data-ng-if="ctrl.services.exchangeStates.constructor.isDeleting(resource)"></span>
-                                <span class="label label-warning"
+                                <span class="oui-status oui-status_warning"
                                       data-translate="exchange_tab_ACCOUNTS_state_SUSPENDED"
                                       data-ng-if="ctrl.services.exchangeStates.constructor.isSuspended(resource)"></span>
-                                <span class="label label-warning"
+                                <span class="oui-status oui-status_warning"
                                       data-translate="exchange_tab_ACCOUNTS_state_SUSPENDING"
                                       data-ng-if="ctrl.services.exchangeStates.constructor.isSuspending(resource)"></span>
-                                <span class="label label-info"
+                                <span class="oui-status oui-status_info"
                                       data-translate="exchange_tab_ACCOUNTS_state_REOPENING"
                                       data-ng-if="ctrl.services.exchangeStates.constructor.isReopening(resource)"></span>
-                                <span class="label label-success"
+                                <span class="oui-status oui-status_success"
                                       data-translate="exchange_tab_ACCOUNTS_state_OK"
                                       data-ng-if="ctrl.services.exchangeStates.constructor.isOk(resource) && resource.taskPendingId === 0"></span>
                             </td>
                             <td>
-                                <button class="btn btn-icon" type="button"
-                                        data-ng-disabled="ctrl.services.exchangeStates.constructor.isCreating(resource)"
-                                        data-linkedpopover="exchange/resource/popover/resource.popover.html"
-                                        data-linkedpopover-placement="left"
-                                        data-linkedpopover-remote="true"
-                                        data-linkedpopover-single="true"
-                                        data-linkedpopover-title="{{'common_actions' | translate}}">
-                                    <span class="oui-icon oui-icon-gear_line oui-icon_small" aria-hidden="true"></span>
-                                </button>
+                                <oui-action-menu data-compact 
+                                                 data-align="end"
+                                                 data-disabled="ctrl.services.exchangeStates.constructor.isCreating(resource) || !ctrl.services.exchangeStates.constructor.isOk(resource)">
```

Comment:
Depending on how both `isCreating` and `isOk` are implemented, maybe you could directly have 
```suggestion
                                                 data-disabled="!ctrl.services.exchangeStates.constructor.isOk(resource)">
```

Reply:
Yes, your right, int this case it's possiblre fixed thx 😄

Reply:
Yes, your right, int this case it's possible fixed thx 😄

Reply:
Yes, your right, int this case it's possible fixed thx 😄